examples/unix_socket.py

```diff
@@ -10,14 +11,15 @@
 async def test(request):
     return response.text("OK")
 
-if __name__ == '__main__':
-    server_address = './uds_socket'
+
+if __name__ == "__main__":
+    server_address = "./uds_socket"
     # Make sure the socket does not already exist
     try:
-            os.unlink(server_address)
+        os.unlink(server_address)
     except OSError:
-            if os.path.exists(server_address):
-                        raise
+        if os.path.exists(server_address):
+            raise
     sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
     sock.bind(server_address)
     app.run(sock=sock)
```

Comment:
Suggest removing this example, or using the built-in UNIX socket support instead.

Reply:
@Tronic you mean by using the `bind_unix_socket` method (from Sanic) or `loop.create_unix_server`?

Reply:
`app.run(unix=....)` unless there is some specific reason not to?

Reply:
TBH, these examples are best served in the documentation. This PR was mainly to get the examples that were broken running again. Going forward I do not intend to continue adding to this section, but for the people that know this exists and may return back to it, we should make sure they are operational. I do not think we need to make it a showcase.

Reply:
@Tronic thanks. there are N ways to get an asyncio server running with UNIX sockets, we just need to stick to one I guess ... after testing which one is the best, of course :sweat_smile: